psi4/driver/p4util/python_helpers.py

```diff
@@ -1553,3 +1553,62 @@ def _core_triplet(A, B, C, transA, transB, transC):
 
 core.Matrix.doublet = staticmethod(_core_doublet)
 core.Matrix.triplet = staticmethod(_core_triplet)
+
+
+@staticmethod
+def _core_erisieve_build(
+        orbital_basis: core.BasisSet,
+        cutoff: float = 0.0,
+        do_csam: bool = False
+    ) -> core.ERISieve:
+    """
+    Constructs a Psi4 ERISieve object from an input basis set, with an optional cutoff threshold for
+    ERI screening and an optional input to enable CSAM screening (over Schwarz screening).
+
+    Parameters
+    ----------
+    orbital_basis
+        Basis set to use in the ERISieve object.
+    cutoff
+        Integral cutoff threshold to use for Schwarz/CSAM screening. Defaults to 0.0, disabling screening entirely.
+    do_csam
+        Use CSAM screening? If True, CSAM screening is used; else, Schwarz screening is used. By default,
+        Schwarz screening is utilized.
+
+    Returns
+    -------
+    ERISieve
+        Initialized ERISieve object.
+
+    Example
+    -------
+    >>> sieve = psi4.core.ERISieve.build(bas, cutoff, csam)
+    """
+
+    warnings.warn(
+        "`ERISieve` is deprecated in favor of `TwoBodyAOInt`, and will be removed as soon as Psi4 v1.9 is released.\n",
+        category=FutureWarning,
+        stacklevel=2)
+
+    factory = core.IntegralFactory(orbital_basis)
+
+    global erisieve_twobody
+    erisieve_twobody = factory.eri(0)
+
+    return core.ERISieve(orbital_basis, cutoff, do_csam)
+
+def _core_erisieve_shell_significant(self, M, N, R, S):
+    """
+    Determine if a given shell quartet (MN | RS) is sigificant or not. Return True if it is, and False if it is not.
+    """
+
+    warnings.warn(
+        "`ERISieve` is deprecated in favor of `TwoBodyAOInt`, and will be removed as soon as Psi4 v1.9 is released.\n",
+        category=FutureWarning,
+        stacklevel=2)
+
+    return erisieve_twobody.shell_significant(M, N, R, S)
+
+
+core.ERISieve.build = _core_erisieve_build
```

Comment:
Probably only the constructor is needed since it must happen before shell_significant can be run, right? But both is fine. You've built one to confirm the warning and hookups?

Btw, I suspect the mac ecosys is caught in the qcf hanging that I introduced yesterday but fixed as of #2946 merge. I'll cancel it, but that shouldn't count against the PR.

Reply:
Ok, I think I'm for removing `shell_significant`, then. (Thanks for the thoroughness in including it, though.) And thanks for the testing. After that, lgtm!

Reply:
> Ok, I think I'm for removing `shell_significant`, then. (Thanks for the thoroughness in including it, though.) And thanks for the testing. After that, lgtm!

Yeah, I was very unhappy with needing a global variable to make it work, so I'm for removing it, as well. I will do so. Thanks!

Reply:
Oh, and either rebase or kill off the mac ecosys, please, to avert wasting CI.